pilot/pkg/networking/plugin/authn/authentication.go

```diff
@@ -89,7 +90,7 @@ func GetMutualTLS(policy *authn.Policy, proxyType model.NodeType) *authn.MutualT
 }
 
 // setupFilterChains sets up filter chains based on authentication policy.
-func setupFilterChains(authnPolicy *authn.Policy) []plugin.FilterChain {
+func setupFilterChains(authnPolicy *authn.Policy, serviceAccount string, meshConfig *meshconfig.MeshConfig) []plugin.FilterChain {
```

Comment:
Can you pass only the value needed, instead of the whole mesh config object.

Reply:
done.